### PR TITLE
Windows Information Protection sunset and Fixed links

### PR DESCRIPTION
**Pull request by martincaddick**
Added news article about Windows Information Protection sunset. Added notes where WIP is mentioned in ABACs and designs to refer back to the news article.

Additional information to be added in future re:
1. Conditional Access + session controls using Microsoft Defender for Cloud Apps
2. Using Windows MAM (Only for unmanaged devices)
3. Leverage Tenant Restrictions to manage how SaaS and LOB applications are accessed
4. Leverage Endpoint DLP to define work vs non-work data boundaries

### CODE REVIEW

**Review by martincaddick — COMMENTED** [review]
Viewed all pages and checked all links

**Review by martincaddick — COMMENTED** [review]
All files reviewed

File: docs/as-built-as-configured/hybrid-task-sequence.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ The following table describes the prerequisites in use in the task sequence.
 
 Item | Description
 --- | ---
-AutopilotConfigurationFile.json | The AutopilotConfigurationFile.json is required to be available within an Microsoft Endpoint Configuration Manager (MECM) package. As a requirement the [Intune enrolment ABAC](../../as-built-as-configured/intune-enrolment) would need to be implemented before this ABAC. 
+AutopilotConfigurationFile.json | The AutopilotConfigurationFile.json is required to be available within an Microsoft Endpoint Configuration Manager (MECM) package. As a requirement the [Intune enrolment ABAC](../as-built-as-configured/intune-enrolment.md) would need to be implemented before this ABAC.
```
Comment (oobemlr) [inline docs/as-built-as-configured/hybrid-task-sequence.md:28]: gtg

File: docs/news/index.md [hunk 2]
```diff
@@ -6,8 +6,9 @@ title: News
 
 |                                                                                                                                                         |
 |---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Novemeber 2023 Update**<br><em>13 Nov 2023</em><br>Windows Information Protection sunset](../news/wip-guidance-2023-11-13.md) |
```
Comment (martincaddick) [inline docs/news/index.md:9]: @oobemlr This has a typo in it

File: docs/news/wip-guidance-2023-11-13.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+**NOTE: This only impacts unmanaged devices**. 
+
+## Where does this leave me?
+
+Your organisation falls into one of three categories:
+1. You have already implemented the blueprint recommendations and you have at least some unmanaged devices, or
+2. You have already implmented the blueprint recommendations and you only have managed devices.
+3. You haven't yet implemented the blueprint recommended configurations
+
```
Comment (SamBH) [inline docs/news/wip-guidance-2023-11-13.md:19]: For consistency, we should have a comma and or after point 2, and finish point 3 with a full stop.

File: docs/news/wip-guidance-2023-11-13.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+### Scenario 2:
+If you are only using WIP with managed devices, Microsoft Endpoint Manager will continue to support WIP with enrollment (managed devices) for the duration of the Operating System (OS) lifecycle and will continue to offer options to enroll both corporate and personal devices for management (and subsequently to receive WIP policy).
+
+**NOTE: There will be no feature updates and at some point Microsoft will drop support for this product. Probably good to add this to your roadmap.**
+
+### Scenario 3:
+
+For all new configurations we recommend using the Alternative Products / Features listed below. More information on the configuration of these is the subject of future work in early 2024.
+
```
Comment (SamBH) [inline docs/news/wip-guidance-2023-11-13.md:31]: Suggest rewording to the following:

For all new blueprint deployments, we recommend using the alternative products/features listed below. More information on how to configure these is planned to be published in early 2024.

File: docs/news/wip-guidance-2023-11-13.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+4. Leverage Endpoint DLP to define work vs non-work data boundaries
+
+### These WIP features do not currently have an alternative
+
+| Feature |  Description | Microsoft Status |
+| --- | --- | --- |
+| Unsanctioned Application Protections | Currently admins can define a set of sanctioned applications, with their own enforcement on activities. This Unsanctioned App Protection feature would extend this to unsanctioned applications as well. | In planning |
+| Context based classification | This set of scenarios involves admins being able to scope protection to the context of where the file originates such as the originating source location or source application of the file. | In planning |
+| Identity | Any protections that rely on the context of the user’s identity including identity separation in Outlook, Edge, and access within enlightened apps. These scenarios will not be supported in Purview. For user level restrictions, our guidance will be to use Tenant Level Restrictions within Azure AD. | Not Planned |
+
```
Comment (SamBH) [inline docs/news/wip-guidance-2023-11-13.md:44]: Entra ID not Azure AD, please. Also lowercase p in planned for consistency.

File: docs/news/index.md (hunk 2 above)
Comment (SamBH) [reply docs/news/index.md:9]: Pending update

**Review by SamBH — CHANGES_REQUESTED** [review]
Please update as per comments.

File: docs/news/wip-guidance-2023-11-13.md (hunk 3 above)
Comment (martincaddick) [reply docs/news/wip-guidance-2023-11-13.md:19]: added , or to the second option

File: docs/news/wip-guidance-2023-11-13.md (hunk 5 above)
Comment (martincaddick) [reply docs/news/wip-guidance-2023-11-13.md:44]: Fixed

**Review by martincaddick — COMMENTED** [review]
Sam these are all resolved.

**Review by SamBH — APPROVED** [review]
